Tests/DeviceDetectorTest.php

```diff
@@ -460,6 +460,25 @@ public function testCheckRegexRestrictionEndCondition(): void
         $this->assertTrue($this->checkRegexRestrictionEndCondition('TestValue(?:[;)/]|$)'), 'pass condition');
     }
 
+    /**
+     * check the Symfony parser for fixtures parsing errors
+     */
+    public function testSymfonyParser(): void
+    {
+        $files       = \array_merge(
+            \glob(__DIR__ . '/../regexes/client/*.yml'),
+            \glob(__DIR__ . '/../regexes/device/*.yml'),
+            \glob(__DIR__ . '/../regexes/*.yml')
+        );
+        $yamlSymfony = new Symfony();
+
+        foreach ($files as $file) {
+            $yamlSymfony->parseFile($file);
+        }
+
+        $this->markTestIncomplete();
```

Comment:
Guess it shouldn't be marked as incomplete, right?
If you want to avoid the test being marked as risky (due to no assertion) you can add a 
`$this->expectNotToPerformAssertions();`

Reply:
done